src/bogo_shim/config.json

```diff
@@ -68,30 +62,13 @@
 
         "ExtraClientEncryptedExtension-TLS-TLS13": "TLS 1.3 server UNIMPLEMENTED",
 
+        "Server-Verify-*-TLS13": "TODO: FIXME - missing deny-list for outdated signature schemes",
+        "Server-VerifyDefault-*-TLS13": "TODO: FIXME - missing deny-list for outdated signature schemes",
```

Comment:
This should be fixed in a follow-up PR. It seems to be an issue in the selection of signature schemes with client auth (?). The test was not enabled before, as it required session resumption